src/signals-mach.c

```diff
@@ -203,12 +194,6 @@ kern_return_t catch_exception_raise(mach_port_t            exception_port,
     HANDLE_MACH_ERROR("thread_get_state", ret);
     uint64_t fault_addr = exc_state.__faultvaddr;
     if (jl_addr_is_safepoint(fault_addr)) {
-#ifdef JULIA_ENABLE_THREADING
```

Comment:
just a check: it seems this is the only code in this PR that is actually removing code corresponding to the  `JULIA_ENABLE_THREADING==true` case.

Reply:
Ah good catch.  That explains the macOS failure.